shared-bindings/analogbufio/BufferedIn.c

```diff
@@ -49,8 +49,8 @@
 //|         length = 1000
 //|         mybuffer = array.array("H", 0x0000 for i in range(length))
```

Comment:
This is more canonical, right?
```suggestion
//|         mybuffer = array.array("H", [0x0000] * length)
```